### PR TITLE
Fix multiple Light House problem 

### PR DESCRIPTION
**Pull request by chengkangzai**
1. Fix where the application do not have a title 
image
2. Fix where the heading is not in sequence 
image

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
-            <Component {...pageProps} />
-        </CookiesProvider>
+        <>
+            <Head>
+                <title>Nak Naksin!</title>
+            </Head>
+            <CookiesProvider>
+                <Component {...pageProps} />
+            </CookiesProvider>
+        </>
```
Comment (nubpro) [inline pages/_app.tsx]: Think we can put <Head> inside <CookiesProvider> instead
Perhaps in the future, we would like to access cookies within it

File: pages/index.tsx [hunk 2]
```diff
@@ -81,9 +81,9 @@ export default function Home() {
                                         <IoNotificationsOutline size={48} />
                                     </div>
                                     <div className="w-5/6">
-                                        <h4 className="text-black items-center xl:text-2xl">
+                                        <h3 className="text-black items-center xl:text-2xl">
                                             Guaranteed notification
-                                        </h4>
```
Comment (nubpro) [inline pages/index.tsx]: Can we also vertical align the text? It will look better on mobile

**Review by nubpro — CHANGES_REQUESTED** [review]
Small nits, & thanks for running the lighthouse test!

File: pages/_app.tsx (hunk 1 above)
Comment (chengkangzai) [reply pages/_app.tsx]: you mean put the Head tag in the Cookies Providers tag?

File: pages/index.tsx (hunk 2 above)
Comment (chengkangzai) [reply pages/index.tsx]: fix in https://github.com/nubpro/nakvaksin/pull/14/commits/c49ef92867cd7f3ee411ff6c0778d46cd26ba1c5

File: pages/_app.tsx (hunk 1 above)
Comment (nubpro) [reply pages/_app.tsx]: ya

File: pages/index.tsx (hunk 2 above)
Comment (nubpro) [reply pages/index.tsx]: Eh vertical align, not horizontal align

Comment (chengkangzai) [reply pages/index.tsx]: image

this is not what you want kah :3

**Comment by chengkangzai** [comment]
In https://github.com/nubpro/nakvaksin/pull/14/commits/c49ef92867cd7f3ee411ff6c0778d46cd26ba1c5, I made some UI Changes 

1. Justify paragraph 
2. Bold the heading
3. Vertical Align heading with icon
4. Center the heading

File: pages/index.tsx (hunk 2 above)
Comment (chengkangzai) [reply pages/index.tsx]: fixed in https://github.com/nubpro/nakvaksin/pull/14/commits/9f4bc3e0c5dc00bc583bf1051b2ecc75a51409f4

File: pages/_app.tsx (hunk 1 above)
Comment (chengkangzai) [reply pages/_app.tsx]: resolve in 
https://github.com/nubpro/nakvaksin/pull/14/commits/48b232c3aee1eaa3e7898161bb84293623706916

**Review by nubpro — APPROVED** [review]
More polish needed in the future, specifically the blue heading

**Comment by nubpro** [comment]
More work to be done to refactor header and the background curvy image (for simplication purposes) in future PR

It's alright to be merged now, thanks @chengkangzai!

**Pull request merged**